starlite/config/openapi.py

```diff
@@ -137,14 +147,19 @@ def to_openapi_schema(self) -> OpenAPI:
             ),
         )
 
-    def create_openapi_schema_model(self, app: "Starlite") -> OpenAPI:
-        """Creates `OpenAPI` instance for the given `app`.
+    def create_openapi_schema_model(self, app: "Starlite") -> "OpenAPI":
+        """Creates instance of.
+
+        [OpenAPI][pydantic_openapi_schema.v3_1_0.open_api.OpenAPI] instance for
+        the given.
+
+            [Starlite][starlite.app.Starlite] application.
```

Comment:
```suggestion
        the given [Starlite][starlite.app.Starlite] application.
```
This must be one of the pre-commit hooks, docformatter maybe??